scripts.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function() {
```

Comment:
This looks really great - clean and eloquent! Keep playing around with it and see what you can come up with.